packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart

```diff
@@ -37,7 +37,7 @@ Future<InstanceRef> _resolveInstanceRefForPath(
         ref.watch(_providerChanged(path.providerId));
 
         return eval.safeEval(
-          'ProviderBinding.debugInstance.providerDetails["${path.providerId}"]!.value',
+          'ProviderBinding.debugInstance.providerDetails["${path.providerId}"]?.value',
```

Comment:
It is risky to assume null safe syntax as apps may not be running in null safe mode. I think this is the right fix for now but there may be another issue probbly in the ProviderBinding.debugInstance.providerDetails code that triggers this stack overflow in some cases
`flutter: [ERROR]: Provider AutoDisposeStateNotifierProvider<StateController<AsyncValue<InstanceDetails>>, AsyncValue<InstanceDetails>>#00fca(name: familyDebounced(0:00:01.000000) (InstancePath.fromProviderId(providerId: 6, pathToProperty: [PathToProperty.objectProperty(name: _uriToScriptMap, ownerUri: package:devtools_app/src/debugger/debugger_controller.dart, ownerName: DebuggerController)])), from: Instance of 'AutoDisposeStateNotifierProviderFamily<StateController<AsyncValue<InstanceDetails>>, AsyncValue<InstanceDetails>, InstancePath>', argument: InstancePath.fromProviderId(providerId: 6, pathToProperty: [PathToProperty.objectProperty(name: _uriToScriptMap, ownerUri: package:devtools_app/src/debugger/debugger_controller.dart, ownerName: DebuggerController)])) failed with "Evaluation `value` failed with [ErrorRef id: objects/35481, kind: UnhandledException, message: Unhandled exception:
Stack Overflow
`